jwt/jwt.js

```diff
@@ -0,0 +1,31 @@
+function generateJwtTest(env, clientToken, privateKey, keyId) {
```

Comment:
we should update the test to include the api version as well

Reply:
probably another ticket if it's the test?

Reply:
Created  a [ticket](https://jira.cms.gov/browse/DPC-1844)